ci/docker/linux-r.dockerfile

```diff
@@ -46,6 +46,10 @@ COPY ci/scripts/install_minio.sh /arrow/ci/scripts/
 COPY ci/scripts/install_gcs_testbench.sh /arrow/ci/scripts/
 RUN /arrow/ci/scripts/r_docker_configure.sh
 
+# Set up Python 3 and its dependencies
+RUN ln -s /usr/bin/python3 /usr/local/bin/python && \
+    ln -s /usr/bin/pip3 /usr/local/bin/pip
+
```

Comment:
Same above, Python(3) was already installed, but it isn't on the path as `python` so the duckdb source install bails